Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM golang:alpine AS builder
```

Comment:
>@apupier
> ```dockerfile
> FROM quay.io/eclipse/che-sidecar-kubernetes-tooling:1.0.9-a40b3bb
> 
> RUN dnf install -y dnf-plugins-core && \
>       dnf copr enable -y chmouel/tektoncd-cli && \
>       dnf install -y tektoncd-cli
> ```



> @benotif yes, good idea about reusing parts of other images.
> Also, as we want to reduce the number of containers when extensions may work in the same container, inheriting from the kubernetes image might help to achieve that.

Reply:
I have not tested this yet

Reply:
Okay let me try the inheritance approach.